opencti-platform/opencti-graphql/src/domain/file.js

```diff
@@ -98,20 +98,22 @@ export const askJobImport = async (context, user, args) => {
     entity_name: entityName,
     entity_type: entityType
   };
-  if (entity.creator_id) {
-    contextData.creator_ids = Array.isArray(entity.creator_id) ? entity.creator_id : [entity.creator_id];
-  }
-  if (entity[RELATION_GRANTED_TO]) {
-    contextData.granted_refs_ids = entity[RELATION_GRANTED_TO];
-  }
-  if (entity[RELATION_OBJECT_MARKING]) {
-    contextData.object_marking_refs_ids = entity[RELATION_OBJECT_MARKING];
-  }
-  if (entity[RELATION_CREATED_BY]) {
-    contextData.created_by_ref_id = entity[RELATION_CREATED_BY];
-  }
-  if (entity[RELATION_OBJECT_LABEL]) {
-    contextData.labels_ids = entity[RELATION_OBJECT_LABEL];
+  if (entity) { // Entity can be null for global
+    if (entity.creator_id) {
+      contextData.creator_ids = Array.isArray(entity.creator_id) ? entity.creator_id : [entity.creator_id];
+    }
+    if (entity[RELATION_GRANTED_TO]) {
+      contextData.granted_refs_ids = entity[RELATION_GRANTED_TO];
+    }
+    if (entity[RELATION_OBJECT_MARKING]) {
+      contextData.object_marking_refs_ids = entity[RELATION_OBJECT_MARKING];
+    }
+    if (entity[RELATION_CREATED_BY]) {
+      contextData.created_by_ref_id = entity[RELATION_CREATED_BY];
+    }
+    if (entity[RELATION_OBJECT_LABEL]) {
+      contextData.labels_ids = entity[RELATION_OBJECT_LABEL];
+    }
```

Comment:
Those lines are duplicates from `domain/stix.js`.
I took the liberty to refactor both files with a dedicated function reducing the nesting of `if` and the code duplication